app/src/main/java/com/example/minwoo/pingpongchat_client/FriendListActivity.java

```diff
@@ -117,12 +117,16 @@ public void onResponse(Call<List<FriendInfo>> call, Response<List<FriendInfo>> r
                         String friendName = FriendInfo.get(i).getFriendName();
                         String friendEmail = FriendInfo.get(i).getfriendEmail();
                         String friendPhoto = FriendInfo.get(i).getFriendPhoto();
+
+                        if(!(Objects.equals(acct.getEmail(), friendEmail)
+                        && Objects.equals(acct.getDisplayName(), friendName))){
```

Comment:
statement indentation should be like this.
1. you don't have to add '()' after '!' if it same logically.
2. && need indent at front.
if (!Objects.equals(acct.getEmail(), friendEmail
    && Objects.equals(acct.getDisplayName(), friendName))){

ref: https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-142311.html